media-libs/libplacebo/libplacebo-4.192.1-r2.ebuild

```diff
@@ -26,11 +26,11 @@ IUSE="glslang lcms +opengl +shaderc test unwind +vulkan"
 REQUIRED_USE="vulkan? ( || ( glslang shaderc ) )"
 RESTRICT="!test? ( test )"
 
-RDEPEND="glslang? ( dev-util/glslang[${MULTILIB_USEDEP}] )
+RDEPEND="glslang? ( <dev-util/glslang-1.3.231:=[${MULTILIB_USEDEP}] )
```

Comment:
Maybe add a comment for why, as the vulkan stuff tends to attract attention?

Reply:
Sure, although I was't planning to let that ebuild live for very long. I just backported quick fixes to it for now.

Reply:
Or well, actually it may need to stay around for some time.